### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by SneL1**
Logo size corrected and new company description added

### CODE REVIEW

File: static_bottom.inc [hunk 1]
```diff
@@ -16,7 +16,7 @@
       </div>
       <div>
         <a href="https://www.snel.com/?utm_source=website&utm_medium=banner&utm_campaign=sponsored_banner&utm_id=grml-sponsored">
-        <img src="/sponsors/Logo-Snel.com.png" width="300" height="70" alt="snel logo"/>
+        <img src="/sponsors/Logo-Snel.com.png" width="300" height="100" alt="snel logo"/>
```
Comment (jkirk) [inline static_bottom.inc:19]: JFTR: the logo has a native resolution of 750x280. The height should then actually be 112 to keep the same aspect ratio.

Comment (SneL1) [reply static_bottom.inc]: I have corrected it to 300x112

**Review by SneL1 — COMMENTED** [review]
Logo image aspect ration changed and company description expanded

**Comment by mika** [comment]
Merged and deployed, thanks for the review, @jkirk!

**Pull request merged**